index.js

```diff
@@ -67,6 +74,50 @@ function toggleColor(element) {
     }
 }
 
+let button = document.getElementById('header');  
+let color = button.style.color;
+button.addEventListener('click', function ( ){
+    color = button.style.color = color === 'black' ? 'red' : 'black';
+});
+
+
 /***** Deliverable 2 *****/
+let heart = document.querySelector(".like-button")
+let inc = document.querySelector(".likes")
+heart.addEventListener("click", function ( ){
+    let numlikes = parseInt(inc.innerHTML) +1
+    inc.textContent = `${numlikes} Likes`
+});
 
 /***** Deliverable 3 *****/
+let form = document.querySelector("#new-animal-sighting-form")
+
+form.addEventListener('submit', event =>{
+    event.preventDefault( )
+    
+    console.log("Form submission has happened")
+        species = document.querySelector("#animal-species")
+        const li = document.createElement("li")
+        const p = document.createElement("p")
+        input = document.querySelector("#sighting-description")
+        p.textContent = input.value 
+    
+        const img = document.createElement("img")
+        img_input = document.querySelector("#animal-photo")
+        img.textContent = img_input.value 
+        img.src = img_input.value 
+        img.alt =document.querySelector("#animal-species")
+
+
+        const a = document.createElement("a")
+        vid_input = document.querySelector("#animal-link")
+        vid_input.textContent = vid_input.value 
+        vid_input.href = vid_input.value 
+        vid_input.target = "_blank"
+    // wasnt able to show the video, could you help me understand how to? g
```

Comment:
For these you're adding the `href` and `target` on the wrong element. Same as my comment above. 

```javascript
a.href = vid_input.value 
a.target = "_blank"
```